src/core/stdc/complex.d

```diff
@@ -1,6 +1,11 @@
 /**
  * D header file for C99.
  *
+ * This contains bindings to selected types and functions from the standard C
+ * header $(LINK2 http://pubs.opengroup.org/onlinepubs/009695399/basedefs/complex.h.html, <complex.h>). Note
+ * that this is not automatically generated, and may omit some types/functions
+ * from the original C header.
```

Comment:
Yep, will do. But I think due to the possible instability of the Internet, I'll take both name of header and URL as parameters.